db/migrate/20230815120123_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
- [ ] According to the project guide you are to reference the user table as author_id, so your migration should be so ` add_reference :comments, :author, foreign_key: { to_table: :users }, null: false`